package.json

```diff
@@ -3,11 +3,15 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "npx nx serve demo",
-    "build:all": "npx nx run-many --targets=build",
-    "verify:all": "npx nx run-many --targets=lint,test,build"
+    "start": "nx serve demo",
```

Comment:
why did you remove `nx`?

Reply:
I take it you mean the removal of `npx`? It's no longer needed as `npx` won't work with pnpm (I think), and is not necessary (I believe).